pkg/registry/batch/cronjob/strategy.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 
 	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
```

Comment:
Should there be an integration or REST test that enforces this?

Reply:
added to unit tests of strategy